lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -149,3 +154,52 @@ $nx-global-sidebar-separator-color: $nx-grey-900;
     visibility: hidden;
   }
 }
+
+.nx-global-sidebar__other-content {
+  @include container-vertical;
+
+  padding-bottom: $nx-spacing-md;
+
+  + .nx-global-sidebar__footer {
+    border-top: 1px solid $nx-global-sidebar-separator-color;
+  }
+}
+
+.nx-global-sidebar__footer {
+  @include container-vertical;
+
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: auto;
+  padding-top: $nx-spacing-xxl;
+  padding-bottom: $nx-spacing-l;
+  text-align: center;
+}
+
+.nx-global-sidebar__support {
+  margin-bottom: $nx-spacing-xxl;
+
+  .nx-text-link {
+    align-items: center;
+    display: flex;
+  }
+}
+
+.nx-global-sidebar__support-text {
+  font-size: $nx-font-size-s;
```

Comment:
Having the icon and the text at different font sizes causes alignment problems.  Also I'm not seeing a mockup for this ticket so I can check what sizes they should be.

Reply:
the mockups under RSC-plaid in sketch cloud for the side nav are the most recently updated and should be referred to for this

Reply:
> Having the icon and the text at different font sizes causes alignment problems. Also I'm not seeing a mockup for this ticket so I can check what sizes they should be.

Current mockup still shows 14px for the text and 16px for the icon: https://www.sketch.com/s/854743a8-74f5-4529-be90-fd703d583732/a/v8EkrRM#Inspector

Reply:
> Current mockup still shows 14px for the text and 16px for the icon: https://www.sketch.com/s/854743a8-74f5-4529-be90-fd703d583732/a/v8EkrRM#Inspector

Ok.  The alignment happens to more or less match the mockup too (and looks good) so this is fine.